src/lib/y2users/linux/writer.rb

```diff
@@ -168,8 +168,28 @@ def write
       CHAGE = "/usr/bin/chage".freeze
       private_constant :CHAGE
 
-      def users_finder
-        @users_finder ||= UsersFinder.new(initial_config, config)
+      # Creates the new users
+      #
+      # @param issues [Y2Issues::List]
+      def add_users(issues)
+        new_users = config.users.without(initial_config.users.ids)
```

Comment:
We discussed about it after one of our daily calls (IIRC, you could not attend). Although adding is a clear operation (i.e., the result is a new collection with elements of both collections), other arithmetic operations like subtracting or diff are not that clear. What should be the result of `users1 - users2`? The problem is that we tend to think that similar to array, the result should be users1 less elements that are equal to any element from users2. But for a ConfigElement, equality only means that to elements have the same values, but they could be different elements (i.e., different ids). So, there are two options for implementing `#-`: a) subtract elements with the same id or b) subtract elements with the same values. Due to this duality, we decided to avoid arithmetic operations. Instead of that, we have more auto-explanatory API like `#without(ids)`, where it is clear that you want a new collection without the indicated ids.